pagopa/io-app
# chore: [IOBP-607] Fix payments landing screen scrollview behavior

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5654

## Short description
This PR refactors the payments landing screen to fix its scrollview behavior.

## List of changes proposed in this pull request
- Refactored components to allow scrollview correct behavior

## How to test
With the _io-dev-api-server_, enable the **New wallet section** from the **Profile** screen. Navigate to the **Payments** landing screen and check that every component is rendered correctly

## Preview

| Before | After |
| --- | ---| 
| <video src="https://github.com/pagopa/io-app/assets/6160324/a578fd93-5f38-4146-8164-912a92a528ea" /> |  <video src="https://github.com/pagopa/io-app/assets/6160324/e3187043-e06d-4faa-a2ce-f6a06b31858a" />
